torch/_inductor/compiler_bisector.py

```diff
@@ -618,8 +687,62 @@ def command_line_usage() -> None:
         bisection_manager.initialize_system()
         sys.exit(0)
 
+    if command == "run":
+        if len(sys.argv) < 3:
+            print(
+                "Usage: python -m torch._inductor.compiler_bisector run <command> [args...]"
+            )
+            sys.exit(1)
+
+        import subprocess
+
+        run_cmd = sys.argv[2:]
+
+        def test_function() -> bool:
+            # Pass bisection state to subprocess via environment variables
+            env = os.environ.copy()
+            backend = bisection_manager.get_backend()
+            subsystem = bisection_manager.get_subsystem()
+
+            if backend:
+                # For test script to select the right backend
+                env["TORCH_COMPILE_BACKEND"] = backend
+                # For bisector in subprocess to know which backend we're testing
+                env["TORCH_BISECT_BACKEND"] = backend
+
+            if subsystem:
+                assert backend is not None  # subsystem requires a backend
+                env["TORCH_BISECT_SUBSYSTEM"] = subsystem
+                # Get run_state to determine TORCH_BISECT_MAX
+                run_state = bisection_manager.get_run_state(backend, subsystem)
+                if run_state == "test_disable":
+                    # -1 means always disable (counter > -1 is always True)
+                    env["TORCH_BISECT_MAX"] = "-1"
+                elif run_state == "find_max_bounds":
+                    # Subprocess can't report count back, so we estimate upper bound
+                    # Run without disabling, then set a reasonable upper bound
+                    bisection_manager.update_bisect_range(backend, subsystem, 0, 1000)
```

Comment:
We should be able to look at the written file: https://github.com/pytorch/pytorch/blob/da04948d63246f710040bf6c0cf84918fa07eb34/torch/_inductor/compiler_bisector.py#L193

Reply:
1000 might be too small - large graphs have a number of input nodes